styles/roblox/catppuccin.user.css

```diff
@@ -0,0 +1,199 @@
+        background-color: @base;
+    }      
+     .item-card-container .item-card-thumb-container, .btr-pager-holder input.btr-pager-cur{
+        border-color: @base
+    }
+      .btn-toggle.on{
+          background: @accent-color;
+      }
+
+      .btr-settings-checkbox input[type="checkbox"] + label:before{
+          border-color: @surface2;
+      }
+      .btr-settings-checkbox input[type="checkbox"]:checked + label:before{
+          background-color: @accent-color;
+      }
+      .btr-settings-enabled-toggle.checked{
+          background: @accent-color;
+      }
+      .highlight-badge{
+          background: @accent-color;
+      }
+
+    .btr-explorer-header, .btr-properties-header, .btr-explorer-filter{
+        border-color: @surface1;
+    }
+    .btr-explorer-has-children .btr-explorer-more::before, .btr-properties .btr-property-group .btr-property-group-more::before{
+        border-left-color: @accent-color;
+    }
+    .btr-explorer-header .btr-dropdown-container .btr-dropdown-menu > li.selected a{
+       background: @overlay2
+    }
+    .btr-explorer-header .btr-dropdown-container .btr-dropdown-menu > li:hover a{
+        background: @overlay1
+    }
+    .notification, .notification-blue, .notification-red, #notifications-bell-badge.bell-red-badge{
+        background-color: @accent-color;
+        color: @base;
+    }
+    #header .btr-nav-notif{
+        background-color: @accent-color;
+        border-color: @accent-color;
+        color: @base;
+    }
+      .btn-growth-md{
+          background-color: @accent-color !important;
+          border-color: @accent-color !important;
+      }
+      .btn-growth-md:hover, .btn-growth-md:active, .rbx-upgrade-now .btn-secondary-md:hover, .rbx-upgrade-now .btn-secondary-md:active, .btn-primary-lg:hover, .btn-primary-lg:active, .btn-primary-xs:hover, .btn-primary-xs:active, .btn-common-play-game-lg:active, .btn-common-play-game-lg:hover, .btn-growth-xs:hover, .btn-growth-xs:active, .btn-growth-sm:hover, .btn-growth-sm:active, .btn-growth-lg:active, .btn-growth-lg:hover, .btn-primary-md:focus{
+          background-color: @accent-color !important;
+          border-color: @accent-color !important;
+          filter: brightness(83%)
+      }
+      .rbx-upgrade-now .btn-secondary-md, .btn-primary-md, .btn-primary-lg, .btn-primary-xs, .btn-common-play-game-lg, .btn-growth-xs, .btn-growth-sm{
+          background-color: @accent-color;
+          border-color: @accent-color;
+      }
+      .shopping-cart-btn-container .shopping-cart-btn .shopping-cart-icon-container .cart-number-container, .pill-toggle input:checked+label, .radio input[type=radio]:checked+label:after, .btr-switch .btr-switch-flip{
+          background: @accent-color;
+      }
+      .btn-growth-lg{
+          background: @accent-color !important;
+          border-color: @accent-color !important;
+      }
```

Comment:
What's the point of having an element with a border of the same color as the background color?

Reply:
I was still learning css when I made this.
Also when I just color the background color, the border is still uncolored (yes for some reason roblox made both the background and border color the same color)

Reply:
Roblox colors both the border and background the same color.

Reply:
What is the point of a border then... :sob: :unamused:

Reply:
> What is the point of a border then... 😭 😒

really have no idea, roblox being roblox

Reply:
On the unmodified Roblox website, that is what they did so I suppose I just followed along with that. With quick testing it does appear that there are some instances where you can see the border color on the button (ex: the play button that appears when hovering over a friend that is playing an experience and you're able to join them) whilst in other instances you can't (ex: the play button on the Experience Page). So I believe it would be best to keep these unchanged to prevent accidental instances of strange coloring.